.github/FUNDING.yml

```diff
@@ -1,3 +1,4 @@
 # Information for Github sponsor button
 
+github: [do1jlr]
```

Comment:
We decided that it is probably the best solution to link to the contribution pages of both websites instead of private accounts to make things more transparent (see #119).

Can you remove your private account again?

Reply:
"Wir" hatten das bei der einführung des spenden-buttons noch anders gehandhabt.

U.a. waren Teile des Vorstands damit einverstanden dies darin zu haben.

Reply:
Grundsätzlich finde ich es besser, wenn die Spenden alle an eine zentrale Stelle gehen, das war schließlich der Grund warum wir die Sache mit dem Förderverein überhaupt angefangen haben, dass es nicht mehr so ein durcheinander gibt. Dachte eigentlich auch dass das deine Meinung ist nach der Nachricht neulich.

Nichtsdesdotrotz hat die Änderung hier auch nix mit der Anderen zu tun, muss also eh in einen anderen PR.

Reply:
Du hast das auch alles in einem PR mit reingeschummelt btw